### PR TITLE
fixed the contains any and not contains any rules

### PR DESCRIPTION
**Pull request by SamGodwin2**
### What is the context of this PR?
There was an issue where answer_values and match_values could be an array or single value depending on the answer being referenced. This change just ensures both cases are handled. 

### How to review 
New stuff works and edited tests pass.

### CODE REVIEW

File: app/questionnaire/rules.py [hunk 1]
```diff
@@ -77,11 +77,11 @@ def evaluate_condition(condition, answer_value, match_value):
         'contains any':
             lambda answer_value, match_value: answer_and_match
             and isinstance(match_value, list)
-            and bool(set(answer_value) & set(match_value)),
+            and bool(set(answer_value) & set(match_value)) if isinstance(answer_value, list) else answer_value in match_value,
```
Comment (heofs) [inline app/questionnaire/rules.py]: I think you forgot to remove about 10 whitespaces on the end of this line. My pylint is complaining

**Comment by SamGodwin2** [comment]
[Enabling 'Any of' rules for routing on radios and numbers (12)](https://trello.com/c/Pib1MNWh/463-enabling-any-of-rules-for-routing-on-radios-and-numbers-12)

File: tests/app/questionnaire/test_rules.py [hunk 2]
```diff
@@ -184,20 +184,20 @@ def test_evaluate_rule_checkbox_contains_all(self):
         self.assertFalse(evaluate_rule(when, []))
 
     def test_evaluate_rule_checkbox_contains_any(self):
-        when = {'values': ['a', 'b'], 'condition': 'contains any'}
+        when = {'values': ['aaa', 'bbb'], 'condition': 'contains any'}
```
Comment (heofs) [inline tests/app/questionnaire/test_rules.py:187]: Maybe throw in a few extra characters here so it tests more cases? For example spaces, apostrophes and question mark?

**Review by heofs — APPROVED** [review]
Looks good to me.

**Review by pricea1 — APPROVED** [review]
Works for me

**Pull request merged**